BDArmory/FX/ExplosionFX.cs

```diff
@@ -232,7 +232,8 @@ private List<BlastHitEvent> ProcessingBlastSphere()
                 //SCRange = (7f * (8.71779789f * 20f * Caliber + 20f * Caliber))* 0.001f; // 5%
                 // Decided to swap it to 10% since 5% gave pretty big ranges on the order of several meters and 10% actually
                 // simplifies down to a linear equation
-                SCRange = (49f * Caliber * 20f) * 0.001f;
+                //SCRange = (49f * Caliber * 20f) * 0.001f;
+                SCRange = 0.12521980673998822299891372544895f * Mathf.Sqrt(penetration - 5f) * Caliber; // Set it to 5 mm of pen instead so that large warheads aren't penalized
```

Comment:
That's an awfully precise value for a float, which can only represent about 7 digits of accuracy. It's even more digits than a 128bit decimal can represent.

Reply:
I just slapped in what came out of the calculator, AFAIK the compiler should round it. I can trim it down to fewer digits if need be though.

Reply:
It's not an issue, I just thought it was funny. The compiler will just truncate it after the first 8.